CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Arbor
+
+First off, thank you for considering contributing to Arbor! It's people like you that
+make Arbor such a great tool. Feel welcome and read the following sections in order to
+know how to ask questions and how to work on something.
+
+## Types of contributions
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the
+documentation, submitting bug reports and feature requests or writing code which can be
+incorporated into Arbor itself.
+
+If you come across a bug in Arbor, please [file an issue on the GitHub issue tracker](https://github.com/arbor-sim/arbor/issues/new).
+You can attach files to issues. Data and bits of code that help to reproduce the issue
+are very welcome; they'll make it easier for Arbor developers to debug and fix your issue.
+
+The documentation is found in the `/doc` subdirectory, C++ examples in `/example` and
+Python examples in `/python/example`. You can add yours in the same way you would contribute
+code, please see the "Github workflow" section.
+
+Please don't use the issue tracker for support questions. You can use the [Arbor Discussions](https://github.com/arbor-sim/arbor/discussions)
```

Comment:
Our policy so far was the exact opposite.

Reply:
On august 10 I announced the change on Slack: "Github just accepted the request for 'Discussions'. I've placed a first post with some explanation, as well as two Issue templates, meant to subtly guide usage-related questions to 'Discussions'. In case someone is bothered by these Issue templates, I can remove them and set it to a blank template (to keep the message about usage-related questions in place)."

Reply:
Thanks for the reminder.